sycl/include/sycl/ext/oneapi/matrix/matrix-tensorcore.hpp

```diff
@@ -182,11 +182,12 @@ template <typename S, typename T,
           sycl::ext::oneapi::experimental::matrix::matrix_use Use,
           size_t NumRows, size_t NumCols,
           sycl::ext::oneapi::experimental::matrix::matrix_layout Layout,
-          access::address_space Space, typename Cond = void>
+          access::address_space Space, access::decorated IsDecorated,
```

Comment:
Tagging @dkhaldi to review changes in `ext/oneapi/matrix` files

Reply:
Also tagging @JackAKirk. Do we need to change any related documentation?

Reply:
Changes in matrix-tensorcore.hpp are fine. I'm not sure it is worth changing any documentation here. I think the upcoming change to the documentation can take account of this change: https://github.com/intel/llvm/pull/6662 .

Reply:
martix-jit-use.hpp should be changed as well.
Tests don't need to change.
do you plan on updating the affected sycl/doc?

Reply:
> martix-jit-use.hpp should be changed as well.

Absolutely! It has been updated and so has the related docs.